core/Measurable/Type/Manager.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+namespace Piwik\Measurable\Type;
+
+use Piwik\Plugin\Manager as PluginManager;
+use Piwik\Measurable\Type;
+
+class Manager
```

Comment:
Renamed it to `TypeManager`. Btw I often just write `Type\Manager` see eg https://github.com/piwik/piwik/commit/1e9e91ba14b6a5b2aa54b15e31fa382a51cadab3#diff-700dbb21e8c2b2cf9dc0a56a71eb86b4L49